src/coveralls.jl

```diff
@@ -92,7 +100,7 @@ module Coveralls
 
             # get the name of the branch if not a pull request
             if get(ENV, "CI_PULL_REQUEST", "false") == "false"
-                data["git"]["branch"] = split(ENV["GIT_BRANCH"], "/")[2]
+                data["git"]["branch"] = String(split(ENV["GIT_BRANCH"], "/")[2])
```

Comment:
I expect the JSON formatting should already handle AbstractString, SubString, and String the same

Reply:
Earlier I was running into issues when it was a `SubString`, but I was debugging, so I had changed a bunch of stuff. Let me get rid of the `String()` here and let's see if it still works.

Reply:
Done.